mkdocs/docs/supplemental-tools.md

```diff
@@ -0,0 +1,46 @@
+# Supplemental Tools
+
+Tank includes several external/supplemental tools that can be used in conjunction with it for a comprehensive load testing experience.
+
+??? tip "Tip: Navigating to the Tools page"
+    Clicking the `Tools` tab takes the user to the main screen, which has links to `downloads` (AND) pages for launching the tools as `java applets`.
+
+There are two sections of links for the tools, `Applets` and `Downlaodable` Tools.
+
+<figure markdown>
+  ![Tools Main page](https://tank.perf.a.intuit.com/docs/html_single/images/tools/tools_1.png){ width="500" }
```

Comment:
Please use the public domain image content

Reply:
Thanks for the comments Athithya. I will address this.